contracts/ToucanRegenBridge.sol

```diff
@@ -90,14 +85,19 @@ contract ToucanRegenBridge is Ownable, Pausable, AccessControl {
     //      Constructor
     // ----------------------------------------
 
-    constructor(address tokenIssuer_, INCTPool nctPool_) Ownable() {
-        tokenIssuer = tokenIssuer_;
+    constructor(
+        INCTPool nctPool_,
+        bytes32[] memory roles,
+        address[] memory accounts
+    ) {
+        require(accounts.length == roles.length, "accounts and roles must have same length");
```

Comment:
Since there is no setter for `nctPool`, we should add a `require` here to ensure the pool is non-zero. Not related to your current changes but probably best to have in a follow-up PR before we deploy.

Reply:
I'll open a quick issue to keep track of it.

Reply:
https://linear.app/toucan/issue/BLU-3756

Reply:
Also a PR https://github.com/regen-network/toucan-bridge/pull/62